apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx

```diff
@@ -144,25 +228,87 @@ function TopValidatorsCard({ state }: { state: ValidatorState }) {
                 headerLabel: 'Address',
                 accessorKey: 'address',
             },
+            {
+                headerLabel: 'Stake',
+                accessorKey: 'stake',
+            },
+            {
+                headerLabel: 'Distribution',
+                accessorKey: 'cumulativeStake',
+            },
             {
                 headerLabel: 'Pubkey Bytes',
                 accessorKey: 'pubkeyBytes',
             },
         ],
     };
+};
+
+function TopValidatorsCardStatic({ limit }: { limit?: number }) {
+    const { data, columns } = validatorsTable(mockState, limit);
+    return <TableCard data={data} columns={columns} />;
+}
+
+function TopValidatorsCardAPI({ limit }: { limit?: number }) {
+    const rpc = useRpc();
+
+    const { data, isLoading, isSuccess, isError } = useQuery(
+        ['validatorS'],
+        async () => {
+            const validatorData: GetObjectDataResponse = await rpc.getObject(
+                VALIDATORS_OBJECT_ID
+            );
+            if (
+                !(
+                    isSuiObject(validatorData.details) &&
+                    isSuiMoveObject(validatorData.details.data)
+                )
+            ) {
+                throw new Error(
+                    'sui system state information not shaped as expected'
+                );
+            }
+            return validatorData.details.data.fields as ValidatorState;
```

Comment:
We probably can separate this into just a GetObject hook, because this really is just that.

Reply:
done